src/main/flight/mixer.h

```diff
@@ -89,6 +89,14 @@ typedef struct mixerConfig_s {
     bool yaw_motors_reversed;
     uint8_t crashflip_motor_percent;
     uint8_t crashflip_expo;
+    uint16_t rpm_limiter_p;
+    uint16_t rpm_limiter_i;
+    uint16_t rpm_limiter_d;
+    uint16_t rpm_limiter_rpm_limit;
+    bool rpm_limiter_acceleration_limiting;
+    uint16_t rpm_limiter_acceleration_limit;
```

Comment:
rpm_limiter_acceleration_limiting and rpm_limiter_acceleration_limit are not used anywhere.
All other RPM limiting stuff here should be covered with #ifdef USE_RPM_LIMITER